Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -90,31 +79,44 @@ The username and password for the user you intend to use must be added to the **
 ## Configure SecurityAndComplianceV2 on Cortex XSOAR
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
-2. Search for **O365 - Security and Compliance - Content Search**.
-3. Authentication / Authorization methods:
-   1. Click **Add instance** to create and configure a new integration instance.
-
-      | **Parameter**   | **Description**                                                   | **Required** |
-      | --------------- | ----------------------------------------------------------------- | ------------ |
-      | url             | Search and Compliance URL.                                         | True         |
-      | App Secret      | The client secret created in Azure.                                | True         |
-      | App ID          | The application (client) ID from Azure                            | True         |
-      | Tenant ID       | The directory (tenant) ID from Azure.                              | True         |
-      | Organization    | The organization name for the Security & Compliance Center.        | True         |
-      | UPN/Email       | The email address (UPN) of the user account for the integration.   | True         |
-      | Insecure        | Trust any certificate (not secure).                                | False        |
-
-   2. Open the War Room in the playground.
-   3. Run the `!o365-sc-auth-start` command and follow the instructions.
-
-   **Expected Output:**
-
-   > ## Security and Compliance - Authorize Instructions
+2. Search for **O365 - Security And Compliance - Content Search v2**.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter**   | **Description**                                                   | **Required** |
+    | --------------- | ----------------------------------------------------------------- |--------------|
+    | UPN/Email       | The email address (UPN) of the user account for the integration.   | True         |
+    | UPN Password    | Password for the specified UPN. Required when using interactive delegated authentication.   | False        |
+    | App ID          | The application (client) ID from Azure                            | False        |
+    | Tenant ID       | The directory (tenant) ID from Azure.                              | False        |
+    | Organization    | The organization name for the Security & Compliance Center.        | False        |
+    | Insecure        | Trust any certificate (not secure).                                | False        |
+
+4. Authentication / Authorization methods:
+
+- App-only (OAuth2.0) Authentication -
+    1. Fill in the UPN, App ID, and Tenant ID parameters in the integration configuration.
+    2. Run the ***o365-sc-auth-start*** command and follow the instructions.
+    3. For testing completion of authorization process run the ***o365-sc-auth-test*** command.
+- Delegated User Authentication -
```

Comment:
Let's also add that some commands, like create action, will only work with this method.
We need to be very clear about it in our docs.